src/main/java/az/javidan/msproduct/model/request/PageCriteria.java

```diff
@@ -12,4 +12,6 @@
 public class PageCriteria {
     private Integer page = 0;
     private Integer count = 5;
+    private String sortBy = "id";
```

Comment:
burda 2 dene enum yaratsan , SortBy(ID, CREATEAD_AT, RATING, PRICE), SortDirection(ASC,DESC).
List<SortingCriteria>
SortingCriteria {
SortBy sortBy;
SortDirection sortDirection,
int priority -> 0,1,22 for example
}